composer.json

```diff
@@ -1,4 +1,45 @@
 {
+  "name": "civicrm/civicrm-core",
+  "description": "Open source constituent relationship management for non-profits, NGOs and advocacy organizations.",
+  "type": "library",
+  "authors": [
```

Comment:
Thanks for drafting this. TBH, though, there've been a lot of contributors, and there's a more complete list in  `CONTRIBUTORS.txt`, `contributor-key.yml`, and `release-notes/*.md`. I'd like to avoid a fourth one. ;) For `composer.json`, maybe just list "CiviCRM LLC" or "civicrm.org ". (If a it really needs human, then maybe the [current core team](https://civicrm.org/teams/core-team).)